doc/howto/cross_compiling/cross_compiling_for_raspberry.md

```diff
@@ -0,0 +1,61 @@
+# Build PaddlePaddle for Raspberry Pi
+You may use any of the following two approaches to build the inference library of PaddlePaddle for Raspberry Pi:
+
+1. Build it: Log in to a Raspberry Pi via SSH and build the library. The required development tools and third-party dependencies are listed in [`/Dockerfile`](https://github.com/PaddlePaddle/Paddle/blob/develop/Dockerfile). 
+
+1. Cross-compile: This article explains how to cross-compile PaddlePaddle for Raspberry Pi on a Linux/x64 computer.
+
+## The Cross-Compiling Toolchain
+
+Step 1. Clone the following Github repo
```

Comment:
Rephrase: Step 1. Clone the Github repo by running the following command.

Reply:
Done